kms/mackms/mackms.go

```diff
@@ -555,6 +555,73 @@ func (*MacKMS) DeleteCertificate(req *apiv1.DeleteCertificateRequest) error {
 	return nil
 }
 
+// SearchKeys searches for keys according to the query URI in the request.
+//
+//   - "" will return all keys managed by the KMS (using the default tag)
+//   - "mackms:" will return all keys managed by the KMS  (using the default tag)
+//   - "mackms:label=my-label" will return all keys using label "my-label" (and the default tag)
+//   - "mackms:hash=the-hash" will return all keys having hash "hash" (and the default tag; generally one result)
+//   - "mackms:tag=my-tag" will search for all keys with "my-tag"
```

Comment:
Would it be possible to search all the keys in the secure enclave that matches the tag/label/hash? 

This `mackms:se=true` or this `mackms:tag=my-tag;se=true` might be useful.

Reply:
I've added support for this in 606fcd9b12dd745665f5469b900de70568d31610. Right now I'm doing the check in the Go code. It might be possible to set the attribute in the query, but it's not so simple to test the actual Secure Enclave operations, so that's why I chose this option.

When `se` is set, its value will be used to filter the results that are returned. If it's not set in the search query, it will be ignored, and both types of keys will be returned.